opentelemetry-zipkin/Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = [
 keywords = ["opentelemetry", "zipkin", "tracing", "async"]
 license = "Apache-2.0"
 edition = "2021"
-rust-version = "1.65"
+rust-version = "1.70"
```

Comment:
Nit - Also update the [Readme.md](https://github.com/open-telemetry/opentelemetry-rust/blob/6aad3dbc994b4554d7cc1d667822c9968f5b0b67/opentelemetry-zipkin/README.md?plain=1#L111). Not related to this PR and to be handled separately, we need to be consistent regarding mentioning or not-mentioning msrv in `Readme.md` for all the otel crates.